sale_order_import_ubl_http/tests/test_sale_order_import_endpoint.py

```diff
@@ -9,7 +9,7 @@
 from odoo.exceptions import UserError
 from odoo.tests.common import SingleTransactionCase
 
-from odoo.addons.sale_order_import_ubl_http.controllers.main import ImportController
+from ..controllers.main import ImportController
```

Comment:
Thanks for the fix.

TBH I hate this... IMO for tests/ folder we should ignore this error